src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ERR.cs

```diff
@@ -29,7 +30,7 @@ internal static partial class Crypto
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_ErrErrorStringN")]
         private static extern unsafe void ErrErrorStringN(ulong e, byte* buf, int len);
 
-        private static unsafe string ErrErrorStringN(ulong error)
+        private static unsafe string? ErrErrorStringN(ulong error)
```

Comment:
In what situation will this return null?  PtrToStringAnsi is being passed a valid pointer; it will not return null.

Reply:
> In what situation will this return null?

Made it nullable because `Marshal.PtrToStringAnsi((IntPtr)buf)` returns nullable

> PtrToStringAnsi is being passed a valid pointer; it will not return null.

Good to know thanks, as `buf` is created within the method it must be valid pointer, will remove `?`